packages/model-eval/src/makeChatLlmEvalCommands.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  ConversationTestCase,
+  EvalConfig,
+  evaluateQuizQuestionAnswerCorrectness,
+  makeEvaluateConversationLastMessageIncludesRegex,
+  makeGenerateLlmConversationData,
+  makeGenerateLlmQuizQuestionAnswer,
+  MakeGenerateQuizDataParams,
+  QuizQuestionTestCase,
+  reportStatsForBinaryEvalRun,
+} from "mongodb-chatbot-evaluation";
+import { ChatLlm } from "mongodb-chatbot-server";
+
+export interface ChatLlmQuizEvalConfig {
+  generatorConfig: MakeGenerateQuizDataParams;
+  name: string;
+}
+
+interface MakeChatLlmQuizEvalCommandsParams {
+  configs: ChatLlmQuizEvalConfig[];
+  quizQuestions: QuizQuestionTestCase[];
+}
+
+export function makeChatLlmQuizEvalCommands({
+  configs,
+  quizQuestions,
+}: MakeChatLlmQuizEvalCommandsParams) {
+  const generateConfig = configs
+    .map((chatLlmEvalConfig) => {
+      const { generatorConfig, name } = chatLlmEvalConfig;
+      return {
+        [name]: {
+          generator: makeGenerateLlmQuizQuestionAnswer(generatorConfig),
+          testCases: quizQuestions,
+          type: "quiz",
+        },
+      };
+    })
+    .reduce(
+      (acc, val) => ({ ...acc, ...val }),
+      {}
+    ) satisfies EvalConfig["commands"]["generate"];
+
+  const evaluationConfig = {
+    quizQuestionCorrect: {
+      evaluator: evaluateQuizQuestionAnswerCorrectness,
+    },
+  } satisfies EvalConfig["commands"]["evaluate"];
+
+  const reportConfig = configs
+    .map((chatLlmEvalConfig) => {
+      const { name } = chatLlmEvalConfig;
+      return {
+        [name]: {
+          reporter: reportStatsForBinaryEvalRun,
+        },
+      };
+    })
+    .reduce(
+      (acc, val) => ({ ...acc, ...val }),
+      {}
+    ) satisfies EvalConfig["commands"]["report"];
+
+  const commands = {
+    generate: generateConfig,
+    evaluate: evaluationConfig,
+    report: reportConfig,
+  } satisfies EvalConfig["commands"];
+  return commands;
+}
+
+export interface ChatLlmConversationsEvalConfig {
+  name: string;
+}
```

Comment:
Do we use this anywhere?

Reply:
looks like no, removing. prob vestige of some previous version